mapiproxy/libmapistore/backends/python/rest.py

```diff
@@ -530,6 +528,19 @@ class PidTagAccessFlag(object):
     AssocContentsTable  = 0x00000020
 
 
+class RelOperator(object):
+    """ Define possible values for property comparison operators
```

Comment:
Please use PEP8: one-line summary for docstrings, no leading or trailing whitespace.